.github/workflows/build-website.yml

```diff
@@ -28,10 +28,6 @@ jobs:
     # Allow failure when website build fails. This won't block merging but the status will still be red allowing for investigations
     continue-on-error: true
     steps:
-    - name: Cancel Previous Runs
-      uses: n1hility/cancel-previous-runs@v2
-      with:
-        token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Would this not improve CI completion because old runs would be cancelled instead of queued?

Reply:
This was supposed to have an effect of cancelling previous running jobs. 

When I inspected some CI runs, it was not actually kicking in and I preferred to remove it for now.